Library/Homebrew/utils/shell_env.sh

```diff
@@ -0,0 +1,71 @@
+keepkeys=(
```

Comment:
Yes I've never used 'shellcheck' before and a nice "gem", :), that has come out of this process...